src/main/java/kitchenpos/products/tobe/domain/Product.java

```diff
@@ -0,0 +1,78 @@
+package kitchenpos.products.tobe.domain;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+import java.util.Objects;
+import java.util.UUID;
+
+@Table(name = "product")
+@Entity
+public class Product {
+    @Column(name = "id", columnDefinition = "binary(16)")
+    @Id
+    private UUID id;
+
+    @Column(name = "name", nullable = false)
+    @Embedded
+    private DisplayedName name;
+
+    @Column(name = "price", nullable = false)
+    @Embedded
+    private ProductPrice price;
+
+    protected Product() {
+
+    }
+    public Product(UUID id, DisplayedName name, ProductPrice price) {
+        this.id = id;
+        this.name = name;
+        this.price = price;
+    }
```

Comment:
```suggestion
    protected Product() {

    }
    
    public Product(UUID id, DisplayedName name, ProductPrice price) {
        this.id = id;
        this.name = name;
        this.price = price;
    }
```

공백도 의미있게 사용해 보면 좋을 것 같아요. :) 마지막에 코드 정렬을 하는 것도 방법일 것 같아요.

Reply:
인텔리제이에서 마지막 commit이 선언부 위에 나오다 보니;; 미처 못챙겼네요! 수정하겠습니다. 😭